docs/source/reference/config-examples.md

```diff
@@ -79,7 +76,8 @@ export GITHUB_CLIENT_ID=github_id
 export GITHUB_CLIENT_SECRET=github_secret
 export OAUTH_CALLBACK_URL=https://example.com/hub/oauth_callback
 export CONFIGPROXY_AUTH_TOKEN=super-secret
-jupyterhub -f /etc/jupyterhub/jupyterhub_config.py
+# send log output to /var/log
+jupyterhub -f /etc/jupyterhub/jupyterhub_config.py &>> /var/log/jupyterhub.log
```

Comment:
Maybe add a comment about &>>? I had to go look that up, hadn't encountered that before :)

Reply:
Perhaps:
`# append log output to log file in /var/log`

Reply:
👍 done

For background: `>>` appends stdout to a file (common), `&>>` is the same after merging stdout and stderr, which is important for us because Python logging happens on stderr.